frontends/web/src/routes/account/send/components/inputs/scan-qr-video.tsx

```diff
@@ -44,7 +44,7 @@ export const ScanQRVideo = ({
       <video
         className={style.qrVideo}
         ref={videoRef}
-        poster="%3Csvg%20xmlns=%22http://www.w3.org/2000/svg%22%20width=%2264%22%20height=%2248%22%3E%3C/svg%3E"
+        poster="data:image,%3Csvg xmlns='http://www.w3.org/2000/svg' width='64' height='48'%3E%3C/svg%3E"
```

Comment:
nit: @NicolaLS I think `data:image` is fine, but I don't know how convert the original svg to your version. Can we not just use encodeURI or encodeURIComponent?

Reply:
Yes, I'm sorry the svg is actually wrong...`encodeURIComponent` is what we want. With the correct uri in the poster attribute the video is only 48 high as the default `object-fit: fill` t[his css property ](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)is also what caused the distortion with the incorrect uri.

It looks like `object-fit: none` also does not do what we want:

![Screenshot_20231101_131658_BitBoxApp DEBUG](https://github.com/digitalbitbox/bitbox-wallet-app/assets/90551797/d2c96965-0131-4cb1-898f-5915f4104c4b)